charts/cluster-autoscaler/README.md

```diff
@@ -287,6 +287,7 @@ Though enough for the majority of installations, the default PodSecurityPolicy _
 |-----|------|---------|-------------|
 | additionalLabels | object | `{}` | Labels to add to each object of the chart. |
 | affinity | object | `{}` | Affinity for pod assignment |
+| annotations | object | `{}` | Annotations to add to the Deployment object. |
```

Comment:
I'm not keen on having a root value adding annotations only to the deployment. This should either be `deployment.annotations` (like `rbac.serviceAccount.annotations`) or apply to all objects in the chart as `additionalLabels` does and maybe move to being called `additionalAnnotations` to match.